packages/appium/lib/cli/extension-command.js

```diff
@@ -552,7 +619,7 @@ class ExtensionCommand {
     await this.config.updateExtension(installSpec, extData);
   }
 
-  /**
+  /**Promise
```

Comment:
typo here?

Reply:
not a typo.  sometimes refactoring in vscode while files are changing in a background process causes mass hysteria.  these were everywhere.  good catch.